kano_settings/system/display.py

```diff
@@ -122,6 +122,13 @@ def set_hdmi_mode(group=None, mode=None):
 
     set_config_value("hdmi_mode", mode)
 
+# flip screen 180
+def set_flip(display_rotate=None):
```

Comment:
I thought he was? set_config_value is from kano_settings.boot_config https://github.com/KanoComputing/kano-settings/blob/rotate/kano_settings/system/display.py#L15